src/com/luxoft/java/course/shevtsov/homework/algorithm/BinarySearch.java

```diff
@@ -0,0 +1,36 @@
+package com.luxoft.java.course.shevtsov.homework.algorithm;
+
+import com.luxoft.java.course.shevtsov.homework.myUtilities.MyUtilities;
+
+public class BinarySearch {
+
+    public static Integer search(int[] ints, int searchNumber) {
+        if (ints.length > 0 && !MyUtilities.ifArrSotredByAsc(ints)) {
```

Comment:
If length is 0, you are not entering inside 'if'

Reply:
yes, it`s a mistake. corrected